aiplatform/api/AIPlatform.Samples/PredictTextExtractionSample.cs

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START aiplatform_sdk_extraction]
+
+using Google.Cloud.AIPlatform.V1;
+using wkt = Google.Protobuf.WellKnownTypes;
+using System;
+using System.Collections.Generic;
+
+// Extractive Question Answering with a Large Language Model
+public class PredictTextExtractionSample
+{
+    public string PredictTextExtraction(
+        string projectId = "your-project-id",
+        string locationId = "us-central1",
+        string publisher = "google",
+        string model = "text-bison@001")
+    {
+        // Initialize client that will be used to send requests.
+        // This client only needs to be created once,
+        // and can be reused for multiple requests.
+        var client = new PredictionServiceClientBuilder
+        {
+            Endpoint = $"{locationId}-aiplatform.googleapis.com"
+        }.Build();
+
+        // Configure the parent resource.
+        var endpoint = EndpointName.FromProjectLocationPublisherModel(projectId, locationId, publisher, model);
+
+        // Initialize request argument(s).
+        var content = @"
+Background: There is evidence that there have been significant changes
```

Comment:
Is this sample different from the one that's supposed to be summarizing text?

If not, then just keep one and you can add more than one region tag to a single sample.

Also, hoping that the different behaviour of the text-bison model with the different input is explained somewhere.

Reply:
Yes, it's slightly different. I'm merely mirroring Java and other samples, so they have this sample as a separate thing, and I think we should do the same in C# for consistency in docs.